test/new-e2e/tests/ndm/snmp/snmp_test.go

```diff
@@ -155,3 +159,40 @@ func (s *snmpDockerSuite) TestSnmp() {
 		assert.Contains(c, metrics, "snmp.sysUpTimeInstance", "metrics %v doesn't contain snmp.sysUpTimeInstance", metrics)
 	}, 5*time.Minute, 10*time.Second)
 }
+
+func (s *snmpDockerSuite) TestSnmpTagsAreStoredOnRestart() {
+	fakeintake := s.Env().FakeIntake.Client()
+	initialMetrics, err := fakeintake.FilterMetrics("snmp.device.reachable",
+		client.WithTags[*aggregator.MetricSeries]([]string{}),
+	)
```

Comment:
`options` are optional.
An empty list of filtering tags should be equivalent to not specifying any list of filtering tags at all.

```suggestion
	initialMetrics, err := fakeintake.FilterMetrics("snmp.device.reachable")
```

Reply:
changed https://github.com/DataDog/datadog-agent/pull/26003/commits/45ae6cc591eabe4ac00ffe4d6ea11cdb9ff89535